fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -603,6 +617,14 @@ public void readFields(DataInput in) throws IOException {
 
         if (Env.getCurrentEnvJournalVersion() >= FeMetaVersion.VERSION_105) {
             dbProperties = DatabaseProperty.read(in);
+            String txnQuotaStr = dbProperties.getOrDefault("transactionQuotaSize", (transactionQuotaSize == -1L)
```

Comment:
I think the logic should be:
```
String txnQuotaStr = dbProperties.getOrDefault("transactionQuotaSize", String.valueOf(Config.max_running_txn_num_per_db));
```

Because if `transactionQuotaSize` does not exist, than we should use the origin `Config.max_running_txn_num_per_db`.
And once `transactionQuotaSize` is set, next time, we can get `transactionQuotaSize` from `dbProperties` directly.